server/services/UsersService.js

```diff
@@ -183,29 +182,14 @@ async function getAllTesters() {
         );
 
         for (let user of currentUsers) {
-            let results = (
-                await db.sequelize.query(`
-                 select
-                   at_name_id,
-                   name as at_name,
-                   active
-                 from
-                   user_to_at,
-                   at_name
-                 where
-                   user_to_at.at_name_id = at_name.id
-                   and user_id=${user.id}
-            `)
-            )[0];
-
-            user.configured_ats = [];
-            for (let result of results) {
-                user.configured_ats.push({
-                    at_name_id: result.at_name_id,
-                    at_name: result.at_name,
-                    active: result.active
-                });
-            }
+            let ats = await db.UserToAt.findAll({
+                where: { user_id: user.id, active: true },
+                include: db.AtName
+            });
+
+            user.configured_ats = ats.map(r => ({
```

Comment:
Nonblocking: Can you make variable `r` more descriptive?

Reply:
yes fixed!